packages/libandroid-selinux/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://selinuxproject.org
+TERMUX_PKG_DESCRIPTION="Android fork of libselinux, an SELinux userland library"
+TERMUX_PKG_LICENSE="Public Domain"
+TERMUX_PKG_MAINTAINER="@termux"
+_TAG_VERSION=14.0.0
+_TAG_REVISION=11
+TERMUX_PKG_VERSION=${_TAG_VERSION}.${_TAG_REVISION}
+TERMUX_PKG_SRCURL=https://android.googlesource.com/platform/external/selinux
+TERMUX_PKG_GIT_BRANCH=android-${_TAG_VERSION}_r${_TAG_REVISION}
```

Comment:
Parameter expansion is usually preferred.
https://devhints.io/bash#parameter-expansions
```suggestion
TERMUX_PKG_VERSION=14.0.0.11
TERMUX_PKG_GIT_BRANCH=android-${TERMUX_PKG_VERSION%.*}_r${TERMUX_PKG_VERSION##*.}
```